docs/user-guide.md

```diff
@@ -19,9 +19,23 @@ Below are simple steps to use this portal:
 
 {% embed url="https://www.youtube.com/playlist?list=PLJH-POb_55z8YYS_qAk_QNBQeiQ2VrtZD" %}
 
+## Compliance Toolkit Landing Page
+
+On the CTK landing page, partners can view the introduction of the CTK portal and watch a video about CTK. Additionally, they can login or register by clicking the respective buttons. The landing page also provides links to other resources such as the GitHub repository, documentation, and partner details.
+
+![CTK Landing Page1](\_images/ctk-landing-page1.png)
+
+Also, it highlights some of CTK's powerful features, like its pre-built test cases, validations, and detailed reporting.
```

Comment:
Additionally, it showcases several robust features of CTK, including its ready-made test cases, validation mechanisms, and comprehensive reporting.